src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -38,11 +37,14 @@ const selectedChannelKeyTab = atom({
 
 function useChatUtilsHook(): UseChatUtilsHook {
   const [chat, setChat] = useRecoilState(chatAtom);
+  const {selectedChannel, isLoadingFetchingChannelDetail} = chat;
+
   const [selectedChannelKey, setSelectedChannelKey] = useRecoilState(selectedChannelKeyTab);
+
   const {localDb, refresh} = useLocalDatabaseHook();
   const navigation = useNavigation();
-  const {selectedChannel, isLoadingFetchingChannelDetail} = chat;
   const [profile] = (React.useContext(Context) as unknown as any).profile;
+  const {anonProfileId, signedProfileId} = useUserAuthHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The order of variable declarations has been changed. While this doesn't impact the functionality of the code, it's generally a good practice to keep a consistent order of declarations for readability and maintainability. In this case, it seems like the original code was grouping hooks together (`useRecoilState`, `useLocalDatabaseHook`, etc.), which can be a logical way to organize the code.

```diff
- const [selectedChannelKey, setSelectedChannelKey] = useRecoilState(selectedChannelKeyTab);
- const {localDb, refresh} = useLocalDatabaseHook();
- const navigation = useNavigation();
- const {selectedChannel, isLoadingFetchingChannelDetail} = chat;
+ const {selectedChannel, isLoadingFetchingChannelDetail} = chat;
+ const [selectedChannelKey, setSelectedChannelKey] = useRecoilState(selectedChannelKeyTab);
+ const {localDb, refresh} = useLocalDatabaseHook();
+ const navigation = useNavigation();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->